lib/modules/minifyConditionalComments.es6

```diff
@@ -0,0 +1,44 @@
+import htmlnano from '../htmlnano';
+import { isConditionalComment } from '../helpers';
+
+// Spec: https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/compatibility/ms537512(v=vs.85)
+// FIXME: the eslint rule should be removed after ESLint fix the issue: https://github.com/eslint/eslint/issues/13826
+// eslint-disable-next-line no-useless-escape
+const CONDITIONAL_COMMENT_REGEXP = /(<!--\[if\s+?[^<>\[\]]+?]>)([\s\S]+?)(<!\[endif\]-->)/gm;
+
+/** Minify content inside conditional comments */
+export default async function minifyConditionalComments(tree, htmlnanoOptions) {
+    // forEach, tree.walk, tree.match just don't support Promise.
+    for (let i = 0, len = tree.length; i < len; i++) {
```

Comment:
As I mentioned in https://github.com/posthtml/htmlnano/issues/16#issuecomment-724136628, `tree.walk` & `tree.match` is not designed to be work with Promise in the first place. Thus I use for loop to traverse through `tree`.